programs/fileio.c

```diff
@@ -1563,6 +1583,15 @@ FIO_determineCompressedName(const char* srcFileName, const char* outDirName, con
     return dstFileNameBuffer;
 }
 
+static size_t FIO_getLargestFileSize(const char** inFileNames, unsigned nbFiles)
+{
+    size_t i; size_t fileSize; size_t maxFileSize = 0;
```

Comment:
minor : it's common to define multiple variables of same type in the same declaration,
such as : 
`size_t i, fileSize, maxFileSize=0;`